codeception.yml

```diff
@@ -5,8 +5,8 @@ paths:
   data: tests/_data
   support: tests/_support
   envs: tests/_envs
+bootstrap: _bootstrap.php
```

Comment:
The old configuration option (placing `bootstrap` under `settings`) was deprecated as of [Codeception 3.1](https://github.com/Codeception/Codeception/blob/3.1/CHANGELOG-3.1.md) so if we want to go down this route the minimum composer version needs to be bumped aswell. Doing this in Codeception `3.0.X` would [break](https://travis-ci.com/craftcms/cms/builds/123672953) the tests because anything under 3.1 wouldn't understand the new `bootstrap: _bootstrap.php` syntax. 

Also, in line with this change, we can probably stop autoloading the fixture classes using `Codeception\Util\Autoload\Autoload` and start using Composer as the [blog post](http://phptest.club/t/bootstrap-deprecations-in-3-0/2196) advises.

@angrybrad Thoughts on bumping to `3.1`?

Reply:
Thanks @gtettelaar!

Reply:
Strangely, the blog post title is "Bootstrap Deprecations in 3.0".

Reply:
@gtettelaar Can't see any reason not to.

Reply:
Cool! 

I'll clean up the unrequired bootstrap files as well as the example suite for `./craft tests/setup` and then this will be ready to go.